### ISSUE TITLE
Foxy skin: Live battery voltage on bottom left battery meter?

### ISSUE THREAD

**Issue by Justin-L-Franks** [maelstrom2001/ArcticFox#384]
Is there any reason why the battery voltage display at the bottom left doesn't show the voltage drop when firing? Both the Classic and Circles skin do, and this is pretty useful.

**Comment by messywrk** [comment]
+1, I also liked the live voltage while firing.

**Comment by maelstrom2001** [comment]
You can use battery voltage line to display it while firing:

![2017 05 17 17 58 12](https://cloud.githubusercontent.com/assets/18537376/26160703/adb27a4a-3b2a-11e7-8b9a-b0dd899b7517.png)   ![2017 05 17 17 58 19](https://cloud.githubusercontent.com/assets/18537376/26160704/adb23ff8-3b2a-11e7-88fb-21c39ac50b03.png)

**Comment by Justin-L-Franks** [comment]
There are already three separate battery meters (#356), I don't want a fourth. I'm asking if there is a technical reason why the battery voltage displayed at the bottom of the Foxy skin behaves differently than on the other skins.

**Comment by maelstrom2001** [comment]
> three separate battery meters

In standby. While firing there will be displayed the lowest battery voltage under load, in runtime, screenshots are above.

> if there is a technical reason why

Why we should make all skins similar to each other? :)

**Comment by Justin-L-Franks** [comment]
> In standby. While firing there will be displayed the lowest battery voltage under load, in runtime, screenshots are above.

Yes, but you have to give up another option to show it. And then you would have four separate battery meters, which is pretty pointless.

> Why we should make all skins similar to each other? :)

Because it's a rather useful feature, which also frees up space for other useful data.

**Comment by maelstrom2001** [comment]
All this is a matter of taste. It is impossible to please everyone. Please do not forget that first of all the firmware is made for personal use. So you can discuss as much as you want, but that does not mean that absolutely all wishes will be fulfilled. I hope for understanding and thank you for using our products.

**Comment by Justin-L-Franks** [comment]
I understand completely. It just seems odd that you would remove a useful feature from the skin that people like. And if you didn't think it was useful yourself, you never would have put it in the other skins in the first place.
